help.md

```diff
@@ -0,0 +1,34 @@
+https://github.com/Automattic/wp-calypso/tree/master/docs/guide
```

Comment:
uhoh. That's not supposed to be there, that's work in progress for better Jetpack docs! I removed it.